README.md

```diff
@@ -189,9 +189,10 @@ To evaluate the model, use script open_flamingo/eval/evaluate.py. For example, t
 
 ```
 python evaluate.py \
---lm_path facebook/opt-1.3b \
---lm_tokenizer_path facebook/opt-1.3b \
---clip_path openai/clip-vit-large-patch14 \
+--lm_path <path to llama model> \
```

Comment:
why do we need this command in the README at all, if it's just a duplicate of the script? can we just link/refer to the script?